### PR TITLE
Add Changelog

### PR DESCRIPTION
**Pull request by flynnduism**
This PR adds a changelog content area under cloud, available at `/cloud/changelog` as per [FIP052](https://www.notion.so/fermyon/FIP-052-Fermyon-Cloud-Changelog-1c945fa490fe4d2996fe85481554b771)

Screen Shot 2023-04-03 at 2 21 50 PM

-- 

@karthik2804 did most of this work - Rhai script, templating and laying out the section. Big thanks to him 🙌🏻 

* adds a listing with a nice timeline effect for showing recent changes in Cloud
* adds two first entries to the changelog, with their click-through post pages
* adds a url to changelog in the Cloud sidebar
* adds a url in Dev homepage
* [todo] follow up with this PR with one on dotcom to add it to the footer from there too

### CODE REVIEW

File: content/cloud/changelog/gh-actions-spin-deploy.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+type= "changelog_post"
+---
+
+Fermyon has recently released a collection of [GitHub Actions]([https://github.com/fermyon/actions](https://github.com/fermyon/actions)) for working with Spin that will empower you to pick up your development speed: 
+
+- `fermyon/actions/spin/setup` - installs the Spin CLI and necessary plugins
+- `fermyon/actions/spin/push` - pushes a Spin application to a registry
+- `fermyon/actions/spin/deploy` - deploys a Spin application to Fermyon Cloud
+
+Using `spin/deploy`, you can now set up a Continuous Deployment pipeline on Fermyon Cloud. Whenever you merge a pull request into your GitHub repository of choice, `spin/deploy` will trigger a new Spin application deployment. You can learn more about the set-up for this process with our [GitHub Actions tutorial]([https://developer.fermyon.com/cloud/github-actions](https://developer.fermyon.com/cloud/github-actions)).
```
Comment (karthik2804) [inline content/cloud/changelog/gh-actions-spin-deploy.md:16]:
```suggestion
Using `spin/deploy`, you can now set up a Continuous Deployment pipeline on Fermyon Cloud. Whenever you merge a pull request into your GitHub repository of choice, `spin/deploy` will trigger a new Spin application deployment. You can learn more about the set-up for this process with our [GitHub Actions tutorial](https://developer.fermyon.com/cloud/github-actions).
```

File: content/cloud/changelog/gh-actions-spin-deploy.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+title = "GitHub Action support for spin/deploy"
+template = "changelog_item"
+date = "2022-03-30T00:22:56Z"
+enable_shortcodes = true
+tags = ["cicd"]
+[extra]
+type= "changelog_post"
+---
+
+Fermyon has recently released a collection of [GitHub Actions]([https://github.com/fermyon/actions](https://github.com/fermyon/actions)) for working with Spin that will empower you to pick up your development speed:
```
Comment (karthik2804) [inline content/cloud/changelog/gh-actions-spin-deploy.md:10]:
```suggestion
Fermyon has recently released a collection of [GitHub Actions](https://github.com/fermyon/actions) for working with Spin that will empower you to pick up your development speed: 
```

File: content/cloud/changelog/gh-actions-spin-deploy.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+title = "GitHub Action support for spin/deploy"
+template = "changelog_item"
+date = "2022-03-30T00:22:56Z"
+enable_shortcodes = true
+tags = ["cicd"]
```
Comment (karthik2804) [inline content/cloud/changelog/gh-actions-spin-deploy.md:5]:
```suggestion
tags = ["CI/CD"]
```

Just a slight preference but feel free to ignore it.

File: templates/home.hbs [hunk 4]
```diff
@@ -144,6 +144,9 @@
                         <h2>Connect</h2>
 
                         <ul class="connect-links">
+                            <li>
+                                <a href="{{site.info.base_url}}/cloud/changelog">Changelog <span class="external"></span></a>
+                            </li>
```
Comment (karthik2804) [inline templates/home.hbs:149]: Not sure if the changelog fits under the "connect" category.  Maybe in a future PR, we can move it to under the learn more maybe for cloud or somewhere in the footer.

File: content/cloud/changelog.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+title = "Fermyon Cloud Changelog"
+template = "changelog"
+date = "2022-01-01T00:00:01Z"
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/cloud/spin-vs-code-extension.md"
```
Comment (karthik2804) [inline content/cloud/changelog.md:5]: This needs to be removed. apologies for letting it slip through.

**Review by karthik2804 — CHANGES_REQUESTED** [review]
Looks great overall! 

Just a couple of nits and fixes.

**Comment by macolso** [comment]
LGTM

**Review by karthik2804 — APPROVED** [review]
LGTM! 

Just the one minor fix and question that can be addressed later too.

File: templates/home.hbs (hunk 4 above)
Comment (flynnduism) [reply templates/home.hbs:149]: Agree - I will create an issue to revisit / remap the footer.

File: content/cloud/changelog.md (hunk 5 above)
Comment (flynnduism) [reply content/cloud/changelog.md:5]: The whole file, or just `extra.url` part?

Comment (karthik2804) [reply content/cloud/changelog.md:5]: Apologies for not being clear the first time. The `extra.url`. The file needs to exist for it to be rendered.

**Pull request merged**